x-pack/plugins/aiops/public/embeddable/embeddable_change_point_chart_component.tsx

```diff
@@ -40,8 +40,9 @@ export interface EmbeddableChangePointChartProps {
    * Last reload request time, can be used for manual reload
    */
   lastReloadRequestTime?: number;
+  /** Identifier to indicate the plugin utilizing the component */
+  embeddingOrigin?: string;
```

Comment:
Nit. would use `embeddableOrigin` for this prop as it's the component/page containing our embeddable.

Reply:
We used `embeddingOrigin` for the server side telemetry tracking so wanted to keep it consistent.